godot-core/src/builtin/aabb.rs

```diff
@@ -102,7 +106,7 @@ impl Aabb {
         let position = self.position.coord_min(b.position);
```

Comment:
One note, you can technically use fully-qualified syntax if you want to make the symmetry explicit:
```suggestion
        let position = Vector3::coord_min(self.position, b.position);
```

Same as with `f32::min(a, b)` for example.

But I'm fine either way, just noted it since I sometimes forget it myself. Choose what you prefer 🙂